src/core/bookmarkmodel.h

```diff
@@ -56,6 +63,7 @@ class BookmarkModel : public QSortFilterProxyModel
 
   private:
     std::unique_ptr<QgsBookmarkManagerModel> mModel = nullptr;
```

Comment:
`= nullptr` not required.

Careful with the combination QObject and unique_ptr. If a parent is set you can get a problem.
There's a QObjectUniquePtr in qgis.h iirc